dm/master/shardddl/optimist_test.go

```diff
@@ -1029,3 +1030,70 @@ func (t *testOptimist) TestOptimistInitSchema(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(is.TableInfo, DeepEquals, ti1) // the init schema is ti1 now.
 }
+
+func (t *testOptimist) TestSortInfos(c *C) {
```

Comment:
```suggestion
func (t *testOptimist) testSortInfos(c *C) {
```

Or remove line 151 because a capitalized testing will be called by testing framework I think